manager/state/raft/raft_test.go

```diff
@@ -949,3 +951,50 @@ func TestStress(t *testing.T) {
 		assert.True(t, find)
 	}
 }
+
+func GetSnapshotMessage(from, to uint64, size int) *raftpb.Message {
```

Comment:
Put this in `manager/state/raft/testutils` instead of duplicating it in two places.